### PR TITLE
Adding e2e-kubevirt workflow to cluster-api-provider-kubevirt

### PR DESCRIPTION
**Pull request by bardielle**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-release-4.7.yaml [hunk 1]
```diff
@@ -46,6 +46,10 @@ tag_specification:
   name: "4.7"
   namespace: ocp
 tests:
+- as: e2e-kubevirt
+  steps:
+    cluster_profile: kubevirt
+    workflow: openshift-e2e-kubevirt>
```
Comment (chenyosef) [inline ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-release-4.7.yaml:52]:
```suggestion
    workflow: openshift-e2e-kubevirt
```

Comment (bardielle) [reply ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-release-4.7.yaml]: :+1:

File: ci-operator/step-registry/ipi/install/kubevirt/install/ipi-install-kubevirt-install-commands.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+if [[ -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
+  echo "OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE is an empty string, exiting"
+  exit 1
+fi
+
+echo "Installing from release ${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}"
+export SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+export PULL_SECRET_PATH=${CLUSTER_PROFILE_DIR}/pull-secret
+export OPENSHIFT_INSTALL_INVOKER=openshift-internal-ci/${JOB_NAME}/${BUILD_ID}
+export HOME=/tmp
```
Comment (chenyosef) [inline ci-operator/step-registry/ipi/install/kubevirt/install/ipi-install-kubevirt-install-commands.sh:18]: HOME is never used

File: ci-operator/step-registry/ipi/install/kubevirt/install/ipi-install-kubevirt-install-commands.sh [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+azure4) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json;;
+gcp) export GOOGLE_CLOUD_KEYFILE_JSON=${CLUSTER_PROFILE_DIR}/gce.json;;
+kubevirt) ;;
+vsphere) ;;
+openstack) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+openstack-vexxhost) export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml ;;
+*) >&2 echo "Unsupported cluster type '${CLUSTER_TYPE}'"
+esac
+
+dir=/tmp/installer
```
Comment (chenyosef) [inline ci-operator/step-registry/ipi/install/kubevirt/install/ipi-install-kubevirt-install-commands.sh:31]:
```suggestion
dir=$(HOME)//installer
```

Comment (bardielle) [reply ci-operator/step-registry/ipi/install/kubevirt/install/ipi-install-kubevirt-install-commands.sh:31]: I tried not to change things from the original commands.sh:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L31

**Pull request closed without merging**